github-for-jira.sd.yml

```diff
@@ -32,12 +32,13 @@ links:
 cleanup: true
 
 resources:
-  - type: redis
-    name: bottleneck
+  - type: redisx
```

Comment:
Do we use the same thing in our local dockers? Do we need to do any updates there?

Reply:
already did.